tests/integration_test_net.go

```diff
@@ -71,7 +71,9 @@ func isPortFree(host string, port int) bool {
 	if err != nil {
 		return false
 	}
-	listener.Close()
+	if err = listener.Close(); err != nil {
+		fmt.Print("failed to close listener:", err)
```

Comment:
nit: panic instead?

Reply:
I was trying to handler errors without modifying the current behaviour, in that regard, a log or print seemed a better approach than panicking, which could lead to an unexpected runtime panic in a non critical situation.

Reply:
This is test infrastructure - nobody will read the log unless the test fails or crashes.